doc/whats_new/upcoming_changes/sklearn.metrics/29151.fix.rst

```diff
@@ -0,0 +1,4 @@
+- :func:`metrics.det_curve` and :class:`metrics.DetCurveDisplay` now return an
+  extra threshold at infinity where the classifier always predicts the negative
+  class i.e. tps = fps = 0.
+  :pr:`29151` by :user:`Arturo Amor <ArturoAmorQ>`.
```

Comment:
Please add a new changelog entry of the `enh` type to document the new `drop_intermediate` option.